.gitignore

```diff
@@ -21,3 +21,4 @@ tmp/
 /js/
 
 Gemfile.lock
+Vendor/
```

Comment:
does this make a difference, since the `/js/` is already ignored. Shouldn't hurt to try

Reply:
> does this make a difference, since the `/js/` is already ignored. Shouldn't hurt to try

thanks will try soon later today